vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/VividusToZephyrExporterApplication.java

```diff
@@ -43,10 +49,4 @@ public static void main(String[] args) throws IOException, JiraConfigurationExce
         ZephyrExporter exporter = ctx.getBean(ZephyrExporter.class);
         exporter.exportResults();
     }
-
-    @Bean("propertyParser")
-    public PropertyParser propertyParser(Properties properties)
```

Comment:
is it ok to change the behavior of `PropertyParser`? how did it work before this change?

Reply:
it's needed changes for obtaining all needed exporter properties

Reply:
still waiting for an answer

Reply:
> how did it work before this change?

Reply:
It worked via putting default properties in application.property file

Reply:
I guess the same changes should be done for Azure DevOps exporter